doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2696,6 +2696,14 @@ Bluetooth samples
 
 .. rst-class:: v3-0-2 v3-0-1 v3-0-0
 
+NCSDK-33915: The :ref:`direct_test_mode` on nRF54H20 asserts during reception tests and sends too few packets during transmission tests.
+
+  **Affected platforms:** nRF54H20
+
+  **Workaround:** Move the :c:func:`errata216_on_wait` static function call from the :c:func:`radio_start` function to the test command handlers in :file:`dtm.c` which enable radio: :c:func:`dtm_vendor_specific_pkt`, :c:func:`dtm_test_receive` and :c:func:`dtm_test_transmit`.
```

Comment:
```suggestion
NCSDK-33915: The :ref:`direct_test_mode` asserts on nRF54H20 devices
  The sample asserts during reception tests and sends too few packets during transmission tests.

  **Affected platforms:** nRF54H20

  **Workaround:** Move the :c:func:`errata216_on_wait` static function call from the :c:func:`radio_start` function to the test command handlers in :file:`dtm.c`, which enable radio: :c:func:`dtm_vendor_specific_pkt`, :c:func:`dtm_test_receive`, and :c:func:`dtm_test_transmit`.
```
These functions don't link to the API docs.

Reply:
Yes, these are internal functions without public API documentation. By principle we should avoid referring to them in the docs, but I think in this case providing customers with a valid workaround has more value than avoiding confusion due to referencing internal functions.

Reply:
fixed